packages/client/e2e/credit.spec.ts

```diff
@@ -145,13 +145,14 @@ test.describe("Credit", () => {
 
       await expect(rfqsPage.getByTestId("quotes").first()).toContainText(
         "$100",
-        { timeout: 5000 },
+        { timeout: 10000 },
```

Comment:
is there any way we can pull these out into constants e.g.
SHORT_TIMEOUT
MEDIUM_TIMEOUT
CRAZY_TIMEOUT_FOR_OPENFIN_BUG

Reply:
I would rather we had constants for these things .. if there is repetition, extract and use common thing (another mantra)